packages/in_app_purchase/ios/Classes/InAppPurchasePlugin.m

```diff
@@ -171,12 +171,14 @@ - (void)finishTransaction:(FlutterMethodCall *)call result:(FlutterResult)result
   if (!transaction) {
     result([FlutterError
         errorWithCode:@"storekit_platform_invalid_transaction"
-              message:[NSString
-                          stringWithFormat:@"The transaction with transactionIdentifer:%@ does not "
-                                           @"exsit. Note that if the transactionState is "
-                                           @"purchasing, the transactionIdentifier will be "
-                                           @"nil(null). And you should not finish this transaction",
-                                           transaction.transactionIdentifier]
+              message:[NSString stringWithFormat:
+                                    @"The transaction with transactionIdentifer:%@ does not "
+                                    @"exsit. Note that if the transactionState is "
+                                    @"purchasing, the transactionIdentifier will be "
+                                    @"nil(null). And you should not finish this transaction. "
```

Comment:
Nit: Sorry, on second thought I think everything after the sentence about transactionIdentifier being null isn't needed here. It's helpful to know but it's not necessarily relevant, and we don't want to clog up the error logs with extraneous info.